src/client/src/rt-theme/index.ts

```diff
@@ -0,0 +1,3 @@
+export { colors } from './colors'
```

Comment:
Do we still need RT-themes and rt-theme?

Reply:
We don't, this is transitory — want to get this merged for @keshaldesai. My next PR addresses this.